Makefile

```diff
@@ -15,7 +15,7 @@ $(call help,make db,initialize the DB and upgrade it to the latest migration)
 db: args?=upgrade head
 db: python
 	@tox -qe dev --run-command 'python bin/make_db'
-	@tox -qe dev  --run-command 'alembic -c conf/alembic.ini $(args)'
+	@tox -qe dev  --run-command 'alembic $(args)'
```

Comment:
The cookiecutter has made `-c conf/alembic.ini` unnecessary and removed it from here